setup.cfg

```diff
@@ -26,8 +26,8 @@ include_package_data = True
 zip_safe = False
 install_requires =
     protobuf >= 4.22
-    grpcio >= 1.37.0
-    grpcio-status>=1.37.0
+    grpcio >= 1.67.0
```

Comment:
@berndverst any objections on bumping the grpc library version? 
This has been updated in the durable task fork and it's a dependency here.